darwin/torch/dataset.py

```diff
@@ -406,12 +404,11 @@ def _map_annotation(self, index: int):
             max_y = np.max([np.max(y_coord) for y_coord in y_coords])
             w = max_x - min_x + 1
             h = max_y - min_y + 1
-            bbox_area = w * h
             # Compute the area of the polygon
+            # TODO fix with addictive/subtractive paths in complex polygons
```

Comment:
Do you have any suggestions? Is `area` useful? If so, we better fix it sooner rather than later